scripts/ci/kubernetes/ci_run_kubernetes_tests.sh

```diff
@@ -93,7 +93,7 @@ function create_virtualenv() {
         --constraint
         "https://raw.githubusercontent.com/${CONSTRAINTS_GITHUB_REPOSITORY}/${DEFAULT_CONSTRAINTS_BRANCH}/constraints-${HOST_PYTHON_VERSION}.txt"
     )
-    if [[ -n ${GITHUB_REGISTRY_PULL_IMAGE_TAG=} ]]; then
+    if [[ ${CI:=} == "true" ]]; then
```

Comment:
```suggestion
    if [[ ${CI:=} == "true" && -n ${GITHUB_REGISTRY_PULL_IMAGE_TAG=} ]]; then
```

Reply:
This should only happen on CI but onlywhen specific version is used to avoid conflicts (PR has more information), Otherwise using constraints is "safer".

This is BTW one Of the main reasons rewriting breeze to Python will help. MacOS (Bash3) treatment of empty array as "undefined" is killing the whole portability idea.